engine/src/osxtheme.mm

```diff
@@ -152,6 +152,8 @@ static void converttomcrect(const Rect &macR, MCRectangle &mcR)
 	case WTHEME_TYPE_TAB:
 		return True;
 		break;
+    default:
+        break;
```

Comment:
@peter-b the only reason I can think is documentation when comparing v other platorms theming but then you are right it should list all enum values... perhaps it was meant to return false?